API/AddUser.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+    ini_set('display_errors', 1);
+    ini_set('error_reporting', E_ALL);
+
+	$inData = getRequestInfo();
+
+	$servername = "localhost";
+	$sv_username = "root";
+	$sv_password = "";
+	$dbname = "myDB";
+
+	// Create connection
+	$conn = new mysqli($servername, $sv_username, $sv_password, $dbname); 	
+	if( $conn->connect_error )
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		if (userExists($conn, $inData['username']) != false)
+		{
+			flush();
+			returnWithError("Username Already Taken");
+		}
+		else
+		{
+			$hashed_password = password_hash($inData["password"], PASSWORD_DEFAULT);
```

Comment:
The password will be hashed before we send it from the frontend, so you can delete this line please.